cvmfs/catalog_mgr_client.cc

```diff
@@ -220,7 +220,7 @@ LoadError ClientCatalogManager::LoadCatalog(
   fetcher_->cache_mgr()->CommitFromMem(ensemble.manifest->certificate(),
                                        ensemble.cert_buf, ensemble.cert_size,
                                        "certificate for " + repo_name_);
-  ensemble.manifest->ExportChecksum(".", 0600);
+  ensemble.manifest->ExportChecksum(checksum_dir, 0600);
```

Comment:
Many thanks!  The `"."` is certainly a mistake.  For writing, we'd need to use the recently introduced `workspace` directory in order to also cover cases with non-standard, read-only cache setups.  It's not straight forward to get the workspace from the `FileSystem` singleton (mountpoint.cc) at this point in the code.  Perhaps the workspace should be passed to the `ClientCatalogManager` when being created by the `MountPoint` object.  I'll think about it.